internal/format/markdown_document.go

```diff
@@ -124,6 +137,7 @@ const (
 
 	documentTpl = `
 	{{- template "header" . -}}
+	{{- template "resources" . -}}
```

Comment:
I'm thinking moving `resources` section below the `providers. Then the flow of the document when you scroll down reads a bit logical, i.e these are the requirements you have to have and you are using these providers which internally you are using these resources from those. What do you think?

This applies to effectively all the formatters.

Reply:
Makes sense to me. My reasoning for placing this above the providers section was mostly ease of access to documentation links.

Reply:
Yes I see your point and that's a valid one and to be honest I'm not sure which one I like better! Both have their own merits.

Reply:
I'm ok with either as well.

Reply:
I prefer it a bit more to move it down the provider then.